package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.2.2",
   "author": "Izaak Schroeder <izaak.schroeder@gmail.com>",
   "scripts": {
-    "prepublish": "./node_modules/.bin/babel -d ./ ./src/",
+    "prepublish": "./node_modules/.bin/babel  -s -d ./ ./src/",
```

Comment:
Any reason not to `babel  -s -d ./ ./src/`?
npm adds `node_modules/.bin` to path for scripts